src/.editorconfig

```diff
@@ -1,6 +1,15 @@
 # Remove the line below if you want to inherit .editorconfig settings from higher directories
 root = true
 
+# Ignore paths
```

Comment:
>Ignore [](http://example.com/codeflow?start=2&length=6)

FYI, I have pushed an upstream version of .editorconfig to SARIF SDK and will delete others. I have tinkered with this file a bit to enforce some standards we can tighten up. We continue to discuss the standard as a team and will continue to adjust.